torch/csrc/utils/tensor_new.cpp

```diff
@@ -330,6 +330,9 @@ Tensor internal_new_from_data(
     tensor = tensor.to(device, inferred_scalar_type, /*non_blocking=*/false, /*copy=*/false);
   }
 
+  // torch.jit.trace will continue to trace out `.to()` instead of `.lift()`, since
```

Comment:
why is it BC breaking?

Reply:
Oh I think it's technically FC-breaking, not BC-breaking?

E.g. you're running an old version of pytorch that consumes a traced `jit` graph that now has an `at::lift` op in it, which old_pytorch_version doesn't know how to handle.